website/docs/r/mobile_network_sim_policy.html.markdown

```diff
@@ -0,0 +1,196 @@
+
+* `tags` - (Optional) A mapping of tags which should be assigned to the Mobile Network Sim Policies.
+
+---
+
+A `slice_configuration` block supports the following:
+
+* `data_network_configuration` - (Required) An array of `data_network_configuration` block as defined below.
+
+* `default_data_network_id` - (Required) The ID of default data network to use if the UE does not explicitly specify it. Configuration for this object must exist in the `data_network_configuration` block.
+
+* `slice_id` - (Required) The ID of the slice that these settings apply to.
+
+---
+
+A `data_network_configuration` block supports the following:
+
+* `allowed_services_ids` - (Required) An array of IDs of services that can be used as part of this SIM policy. The array must not contain duplicate items and must contain at least one item.
+
+* `data_network_id` - (Required) The ID of Mobile Network Data Network which these settings apply to.
+
+* `session_aggregate_maximum_bit_rate` - (Required) A `session_aggregate_maximum_bit_rate` block as defined below.
+
+* `allocation_and_retention_priority_level` - (Optional) Default QoS Flow allocation and retention priority (ARP) level. Flows with higher priority preempt flows with lower priority, if the settings of `preemption_capability` and `preemption_vulnerability` allow it. 1 is the highest level of priority. If this field is not specified then `qos_indicator` is used to derive the ARP value. See 3GPP TS23.501 section 5.7.2.2 for a full description of the ARP parameters.
+
+* `additional_allowed_session_types` - (Optional) Allowed session types in addition to the default session type. Must not duplicate the default session type. Possible values are `IPv4` and `IPv6`.
+
+* `default_session_type` - (Optional) The default PDU session type, which is used if the UE does not request a specific session type. Possible values are `IPv4` and `IPv6`. Defaults to `IPv4`.
+
+* `qos_indicator` - (Optional) The QoS Indicator (5QI for 5G network /QCI for 4G net work) value identifies a set of QoS characteristics that control QoS forwarding treatment for QoS flows or EPS bearers. Recommended values: 5-9; 69-70; 79-80. Must be between `1` and `127`. Defaults to `9`.
+
+* `max_buffered_packets` - (Optional) The maximum number of downlink packets to buffer at the user plane for High Latency Communication - Extended Buffering. Defaults to `10`, Must be at least `0`, See 3GPP TS29.272 v15.10.0 section 7.3.188 for a full description. This maximum is not guaranteed because there is a internal limit on buffered packets across all PDU sessions.
+
+* `preemption_capability` - (Optional) The Preemption Capability of a QoS Flow controls whether it can preempt another QoS Flow with a lower priority level. See 3GPP TS23.501 section 5.7.2.2 for a full description of the ARP parameters. Possible values are `NotPreempt` and `MayPreempt`, Defaults to `NotPreempt`.
```

Comment:
grammar?
```suggestion
* `preemption_capability` - (Optional) The Preemption Capability of a QoS Flow control and whether it can preempt another QoS Flow with a lower priority level. See 3GPP TS23.501 section 5.7.2.2 for a full description of the ARP parameters. Possible values are `NotPreempt` and `MayPreempt`, Defaults to `NotPreempt`.
```